website/content/docs/secrets/pki/considerations.mdx

```diff
@@ -1,10 +1,11 @@
 ---
 layout: docs
-page_title: 'PKI - Secrets Engines: Considerations'
-description: The PKI secrets engine for Vault generates TLS certificates.
+page_title: 'PKI secrets engine considerations'
+description: >-
+   Understand the important considerations and guidance before using the PKI secrets engine to generate the CA certificates. 
```

Comment:
Suggestion: change to "generate certificates", or even just change to end with "...before using the PKI secrets engine"